apps/evm-bridge/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "iota-evm-bridge",
+    "version": "0.0.0",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc && vite build",
+        "preview": "vite preview",
+        "eslint:check": "eslint --max-warnings=0 \"./src/**/*.{js,jsx,ts,tsx}\"",
+        "eslint:fix": "pnpm eslint:check --fix",
+        "prettier:check": "prettier -c --ignore-unknown --ignore-path=.prettierignore .",
+        "prettier:fix": "prettier -w --ignore-unknown --ignore-path=.prettierignore .",
+        "lint": "pnpm eslint:check && pnpm prettier:check",
+        "lint:fix": "pnpm eslint:fix && pnpm prettier:fix",
+        "test:prepare:L1": "pnpm -w wallet build",
+        "test:prepare:L2": "L1=false vite-node scripts/downloadWallet.js",
```

Comment:
Since we removed the "double script" in theory we don't need this `L1=false` anymore.